<!DOCTYPE html>
<html lang="en-us">
    <head>
    
    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-140792770-1"></script>
    <script>
        window.dataLayer = window.dataLayer || [];
        function gtag() { dataLayer.push(arguments); }
        gtag('js', new Date());

        gtag('config', 'UA-140792770-1');
    </script>

    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

    <title>
        Mapping Kinases and Viral Drug Connectivity &middot; Michael Zhang
    </title>

    
    <script type="text/javascript" async
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    displayMath: [['$$','$$']],
    processEscapes: true,
    processEnvironments: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre'],
    TeX: { equationNumbers: { autoNumber: "AMS" },
         extensions: ["AMSmath.js", "AMSsymbols.js"] }
  }
  });
  MathJax.Hub.Queue(function() {
    
    
    
    var all = MathJax.Hub.getAllJax(), i;
    for(i = 0; i < all.length; i += 1) {
        all[i].SourceElement().parentNode.className += ' has-jax';
    }
  });

  MathJax.Hub.Config({
  
  TeX: { equationNumbers: { autoNumber: "AMS" } }
  });
</script>

    
    <link rel="stylesheet" href="/css/style.css">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Libre+Baskerville:400,400i,700">
    <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet">
    
    <link rel="stylesheet" type="text/css" href="/css/life_contributions.css">

    
    <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-mz.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-mz.png">
    <link rel="apple-touch-icon" sizes="180x180" href="/images/favicon-mz.png">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">

    
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css"
        integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">

    
    <link href="" rel="alternate" type="application/rss+xml" title="Michael Zhang" />

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.12.0/moment.min.js" charset="utf-8"></script>
    <script src="https://d3js.org/d3.v4.min.js" charset="utf-8"></script>
</head>
    <body>
        		<nav class="nav">
			<div class="nav-container">
				<a href="/">
					<h2 class="nav-title">Michael Zhang</h2>
				</a>
				<ul>
    <li><a href="/about">About</a></li>
    <li><a href="/posts">Posts</a></li>
    <li><a href="/projects">Projects</a></li>
    <li><a href="/">Home</a></li>
</ul>
			</div>
		</nav>

        

<main>
	<div class="post">
		<div class="post-info">
    <span>Updated</span><span> on&nbsp;</span><time datetime="2017-01-25 03:38:07 &#43;0900 &#43;0900">January 25, 2017</time>
        

    
</div>
		<h1 class="post-title">Mapping Kinases and Viral Drug Connectivity</h1>
<div class="post-line" style="padding-bottom: 16px"></div>


		
<script type="text/javascript" async
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    displayMath: [['$$','$$']],
    processEscapes: true,
    processEnvironments: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre'],
    TeX: { equationNumbers: { autoNumber: "AMS" },
         extensions: ["AMSmath.js", "AMSsymbols.js"] }
  }
  });
  MathJax.Hub.Queue(function() {
    
    
    
    var all = MathJax.Hub.getAllJax(), i;
    for(i = 0; i < all.length; i += 1) {
        all[i].SourceElement().parentNode.className += ' has-jax';
    }
  });

  MathJax.Hub.Config({
  
  TeX: { equationNumbers: { autoNumber: "AMS" } }
  });
</script>

		<div style="margin-bottom: 2em">
			<i>Changing gene expression to beat out viral infections</i>
			
		    <div class="tag-container">
		    	Tags: [
		        
		         <span class="tag">
		            <a href="https://michaelzhang.xyz/tags/biology">
		                
		        <span class="tag">
		            <a href="https://michaelzhang.xyz/tags/biology">
		                biology
		            </a>
		        </span> 
		        ]
		    </div>
		    
		</div>
		
	    

		

<p><figure>
    <img src="/images/003-kinase-02.png"/> 
</figure>

<em>Source: <a href="http://maillardlab.org/mechanisms-of-signal-transduction-in-pka/">Maillard Lab</a></em></p>

<p>Kinases are pretty interesting to me. Sure I didn&rsquo;t have the best introduction with them - as with most proteins, they were portrayed in AP Biology as either overly simple cartoon squiggly shapes or hopelessly complex ribbon / space-filling models, playing some large role in cell signaling with an equally large number of chemicals and sub-products to memorize. But over time I was able to see past these offenses to arrive at my current sentiment: kinases are kind of cool. Not only do they phosphorylate proteins, playing a role in regulating most aspects of cell life, but abnormal phosphorylation is often the cause or consequence of disease). Accordingly, they&rsquo;ve become an important group of drug targets that I wanted to take a look at.</p>

<p>In an attempt to unearth some computational biology skills, review some recent Pandas exposure, and get back into the spirit of biological research and discovery, I wanted to explore the relationship between kinase differential expression and viral infection. To do so, I ran a general query on the Gene Expression Atlas for all experiments that had to deal with viruses and for genes somehow related to kinases. Following this, the ultimate goal was to use connectivity mapping to find possible drug candidates correlated with a reversion in differential expression.</p>

<h2 id="some-background-motivation">Some Background + Motivation</h2>

<p>Recently, one of five 2016 Breakthrough Prizes in the Life Sciences was awarded to Professor Stephen Elledge, who among other incredible things is particularly known for his work in eludicating the human DNA damage response (DDR). Excitingly, this can basically be summarized as a protein kinase cascade, and so one could imagine directing their drug discovery search in regards to diseases that heavily involve DNA damage towards kinase inhibitors or activators. Cancers are probably what come to mind the most, and I had also done some Pandas notebook exploration into that, but many cells also undergo virus infection-associated DDR.</p>

<h3 id="picking-the-right-virus-infection-targets">Picking the Right Virus Infection Targets</h3>

<p>Another motivating idea is that going after viruses is difficult, particularly because of their rapid mutation cycle frequently rendering drugs obsolete. While we typically think of targeting foreign proteins such as external envelope proteins, another path to take might involve looking elsewhere from the virus entirely, and instead at host targets. While this sounds kind of like targeting your own cells with a therapeutic meant to kill viruses, viral infection might cause the upregulation of certain genes in host cells that allow them to propogate. If we can disrupt these then, the virus has a much harder time of getting by, and can&rsquo;t quite rely on its typical mutation frequency to escape dependency as fast. The idea is that drugs that disrupt host pathways that the virus relies on can&rsquo;t be as easily overcome by the virus through mutation because that would require the virus to evolve an entirely different functionality, which is a much lower probability event then changing a few coat protein residues.</p>

<h3 id="even-more-motivation">Even More Motivation</h3>

<p>Along with talking to <a href="https://www.deshawresearch.com/people_c-b_taube.html">Dr. Andrew Taube at DESRES</a> and learning about their group&rsquo;s work in MD with kinases, phosphatases, and viruses, and having my own probably far-fetched ideas about one day being able to engineer a sort of hyper DDR and immunity against the introduction of viral DNA, I was interested in seeing what I could find through a short exploration with publicly available datasets.</p>

<p>I had also just taken MCB60: Cellular Biology and Molecular Medicine, which involved a lab component exploring DDR in yeast (slides viewable here). Three months of intermittent web lab work is kind of naturally inconclusive, so I was eager to see if I could get some more definite results computationally.</p>

<p>While in the short time putting this together I could only get as far as generating possibly connected drugs that would work towards reversing host cell expression after virus infection, and there&rsquo;s still a ton of analysis to be done with more time, the exercise was pretty fun and interesting. The later parts of the notebook could also definitely be cleaned up. See the <a href="https://michaelzhang.xyz/Kinase-Virus_Analysis/">original notebook and text here</a>.</p>


		
<div class="icons">
  <a href="https://github.com/mzio" target="_blank" class='icon'>
    <i class="fab fa-github icon"></i>
  </a>
  <a href="https://linkedin.com/in/michaelzhangxyz/" target="_blank" class='icon'>
    <i class="fab fa-linkedin-in icon"></i>
  </a>
  <a href="mailto:michael_zhang@college.harvard.edu" class='icon'>
    <i class="fas fa-envelope icon"></i>
  </a>
</div>



	</div>
</main>


        		<footer>
			<span>
			&copy; <time datetime="2021-08-12 12:26:01.977892 -0700 PDT m=&#43;0.162200344">2021</time>. Michael Zhang. H' 2020. 
			</span>
			<div class="container" style="padding-top: 5px; height: 200px; width: 71%;"></div>
			
			<script src="/js/life_contributions.js"></script>
			<script src="/js/life_contributions_commits.js"></script>
			<script type="text/javascript">
				var now = moment().endOf('day').toDate();
				var yearAgo = moment().startOf('day').subtract(1, 'year').toDate();
				var graph = lifeContributions()
				.data(commits)
				.selector('.container')
				.tooltipEnabled(true)
				.onClick(function (data) {
					console.log('data', data);
				});
				graph();  
			</script>
		</footer>

    </body>
</html>
